File: cfgs/pythoncfgs/MINIMUM_FLIP_REQUIRED_MAKE_BINARY_MATRIX_SYMMETRIC_1.py
Nodes
1;statement;foo_1 = 0;8;
2;conditional;fun_1(foo_1);9;
3;statement;return foo_1;13;
4;conditional;fun_1(foo_1);10;
5;conditional;foo_1[foo_2][foo_3] != foo_4[foo_5][foo_6];11;
6;statement;foo_1 += 1;12;
7;exit;;
Edges
0;2;
1;3;4;
2;7;
3;2;5;
4;4;6;
5;4;
6;

File: cfgs/java_cfgs_foo/MINIMUM_FLIP_REQUIRED_MAKE_BINARY_MATRIX_SYMMETRIC_1.java
Nodes
1;special;;;
2;statement;int flip = 0;14;
3;statement;int i = 0;15;
4;conditional;foo_1 < foo_2;16;
5;statement;int j = 0;17;
6;conditional;foo_1 < foo_2;18;
7;conditional;foo_1[foo_2][foo_3] != foo_4[foo_5][foo_6];19;
8;statement;flip++;19;
9;statement;j++;19;
10;statement;i++;17;
11;statement;return foo_1;20;
12;exit;;;
Edges
0;2;
1;3;
2;4;
3;5;11;
4;6;
5;7;10;
6;8;9;
7;9;
8;6;
9;4;
10;12;
11;
